{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/settings.py

```diff
@@ -369,23 +359,43 @@ def _env_get_required(setting_name):
             "filters": ["require_debug_true"],
             "class": "logging.StreamHandler",
         },
-        "rollbar": {
-            "level": "WARNING",
-            "filters": ["require_debug_false"],
-            "access_token": _env_get_required("ROLLBAR_ACCESS_TOKEN"),
-            "environment": ENVIRONMENT,
-            "class": "rollbar.logger.RollbarHandler",
-        },
     },
     "loggers": {
-        "django": {"handlers": ["console", "rollbar", ], "level": "INFO"},
+        "django": {"handlers": ["console", ], "level": "INFO"},
         # The logger name matters -- it MUST match the name of the app
-        "{{ cookiecutter.project_slug }}": {"handlers": ["console", "rollbar",], "level": "DEBUG", "propagate": True},
+        "{{ cookiecutter.project_slug }}": {"handlers": ["console", ], "level": "DEBUG", "propagate": True},
         "{{ cookiecutter.project_slug }}.request": {"handlers": [], "level": "INFO", "propagate": True},
         "{{ cookiecutter.project_slug }}.tasks": {"handlers": [], "level": "INFO", "propagate": True},
     },
 }
 
+#
+# Rollbar logging config
+#
+ROLLBAR_ACCESS_TOKEN = _env_get_required("ROLLBAR_ACCESS_TOKEN") if IN_PROD else os.environ.get("ROLLBAR_ACCESS_TOKEN")
+
+if ROLLBAR_ACCESS_TOKEN:
+    MIDDLEWARE += ["rollbar.contrib.django.middleware.RollbarNotifierMiddleware"]
+    ROLLBAR = {
+        "access_token": ROLLBAR_ACCESS_TOKEN,
+        "environment": ENVIRONMENT,
+        "branch": "main",
```

Comment:
Is there a reason to include the branch name here? This could be wrong if we use this for staging or review apps, right?

Reply:
I pulled this in from another project that had it. Not sure what it does and haven't tested it.

Reply:
docs are a little confusing on this. On the one hand they say if you don't specify this it defaults to `master`.
But then they say `The branch field is optional, but must be set whenever the value is not master or main.`
So perhaps it's best to just remove for now and see if we run into an issue at some point with Review Apps.

Reply:
Yeah, I think this is just metadata (like the environment name) that shows up in the Rollbar dashboard that you can use to sort and filter Rollbar items. If it's always 'main' then it won't be useful, so probably better to remove it here, rather than send (likely) incorrect metadata.